Makefile

```diff
@@ -887,8 +887,6 @@ jslint-ci:
 
 CPPLINT_EXCLUDE ?=
 CPPLINT_EXCLUDE += src/node_root_certs.h
-CPPLINT_EXCLUDE += src/queue.h
```

Comment:
Just for context, `queue.h` was removed in https://github.com/nodejs/node/commit/7e779b459367f071366699550ea0162b63516564 (https://github.com/nodejs/node/pull/667)